s3torchconnector/src/s3torchconnector/dcp/s3_file_system.py

```diff
@@ -283,6 +283,7 @@ def __init__(
             s3client_config (Optional[S3ClientConfig]): Optional S3ClientConfig with parameters for S3 client.
             prefix_strategy (Optional[S3PrefixStrategyBase]): Optional strategy for generating S3 prefixes to
                 optimize checkpoint organization and prevent throttling.
+            thread_count (int): Number of IO threads to use to write. We default to 1 emulating Pytorch defaults
```

Comment:
nit: maybe just write `Defaults to 1`, and optionally add (PyTorch default) / (Parent class default)
```suggestion
            thread_count (int): Number of IO threads to use to write. Defaults to 1 (PyTorch default).
```